platforms-wg/README.md

```diff
@@ -1,5 +1,13 @@
 # Platforms WG
 
+## Note:
+
+This working group has been archived and replaced by the Platform Engineering Community Technical Community Group.
```

Comment:
```suggestion
This working group has been archived and replaced by the [Platform Engineering Community Technical Community Group](https://github.com/cncf/communitygroups/issues/399).
```

Reply:
I guess by the time the community page is up, we should have a link to the GitHub issue.